### PR TITLE
27 Create Nav Bar

### LINKED ISSUE #27: Create navbar component
**Description:**

- Add nav bar to home page.
- Add icons for Account, Settings, and Logout.
- BisonCoin logo will navigate user to home page.

**Acceptance Criteria:**

- [x] Account, Settings, and Logout icons are visible on home page.

### LINKED ISSUE #76: Refactor style variables into style.scss
**Description:**

- Move css variables into style.scss for clean up and reuse.

### PR DESCRIPTION
**Pull request by madisonfines**
Added navigation bar to home page. It includes an account icon, settings icon, and logout icon. Currently, clicking on icons does not navigate user anywhere because those pages will be added in future issues. 

This PR also cleans up the CSS in every Vue component. Creates variables in style.css.

Closes #27 and #76 

**Home page with nav bar:**
Screen Shot 2021-03-03 at 3 35 25 PM

### CODE REVIEW

File: src/frontend/src/App.vue [hunk 1]
```diff
@@ -9,3 +9,13 @@ import { Vue } from "vue-property-decorator";
 
 export default class App extends Vue {}
 </script>
+
+<style lang="scss">
+@import "./style.scss";
+
+body {
```
Comment (ZackHolmberg) [inline src/frontend/src/App.vue:16]: Can you add the new font style in this PR too please:
```
@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;400&display=swap");
body {
  font-family: "Montserrat", sans-serif;
}
```

File: src/frontend/src/pages/HomePage.vue [hunk 2]
```diff
@@ -1,19 +1,24 @@
 <template>
+<div>
+  <NavBar />
   <div class="wrapper">
     <Wallet class="wallet-component"/>
     <CreateTransaction class="transaction-component"/>
   </div>
+  </div>
```
Comment (ZackHolmberg) [inline src/frontend/src/pages/HomePage.vue]: Indentation off here

**Review by ZackHolmberg — APPROVED** [review]

File: src/frontend/src/App.vue (hunk 1 above)
Comment (madisonfines) [reply src/frontend/src/App.vue:16]: Added!

File: src/frontend/src/pages/HomePage.vue (hunk 2 above)
Comment (madisonfines) [reply src/frontend/src/pages/HomePage.vue]: Fixed!

**Review by abhisachdev17 — APPROVED** [review]
LGTM

**Pull request merged**